invenio_app_ils/ui/src/semantic-ui/site/collections/table.overrides

```diff
@@ -1,3 +1,25 @@
 /*******************************
          Site Overrides
 *******************************/
+
+.front-app{
```

Comment:
I am ok with `front-app`, just make sure that we use it consistently everywhere and it does not become `frontsite` from time to time :)